### PR TITLE
Document how to run Nice and Postgres locally in Docker

### PR DESCRIPTION
**Pull request by pgerber**
### Contributor checklist

- [x] My contribution is **ready to be merged** as is
- [x] I verified the modifications **render properly**
- [x] I used **spell checking**
- [x] I **used the styling and structure aids available** in [Sphinx/ResT](http://www.sphinx-doc.org/en/stable/rest.html). (Links use `` `…`_``, enumerations `#.`, lists `*`, code ``` ``…`` ```/`.. code::`, warnings .. `warning::`, etc.)
- [x] I reread the text keeping in mind that the text has to be **comprehended** fully **by the target audience**

Closes: #124

### CODE REVIEW

File: devops/openshift/nice2_and_docker.rst [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+Running the Image Locally
+-------------------------
+
+.. parsed-literal::
+
+    docker run --rm -p 8080:8080 -e NICE2_HIKARI_dataSource.serverName=\ **${DB_SERVER}**
+      -e NICE2_LOGBACK_CONFIG=\ **logback_terminal** -e NICE2_HIKARI_dataSource.databaseName=\ **${DB_NAME}** \
+      -e NICE2_HIKARI_dataSource.user=\ **${DB_USER}** -e NICE2_HIKARI_dataSource.password=\ **${DB_PASSWORD}** \
+      -e NICE2_JAVA_OPT\_-Dch.tocco.nice2.runenv=\ **development** **${DOCKER_IMAGE_NAME}**
```
Comment (rzueger) [inline devops/openshift/nice2_and_docker.rst]: NICE2_HIKARI_dataSource__sslMode=required

File: devops/openshift/nice2_and_docker.rst [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+In order to run DB refactoring, which will create or update a DB, run the same command as for starting the container
+but execute the ``dbref`` command within the container.
+
+.. parsed-literal::
+
+    docker run --rm -p 8080:8080 -e NICE2_HIKARI_dataSource.serverName=${DB_SERVER}
+      -e NICE2_LOGBACK_CONFIG=logback_terminal -e NICE2_HIKARI_dataSource.databaseName=${DB_NAME} \
+      -e NICE2_HIKARI_dataSource.user=${DB_USER} -e NICE2_HIKARI_dataSource.password=${DB_PASSWORD} \
+      -e NICE2_JAVA_OPT\_-Dch.tocco.nice2.runenv=development ${DOCKER_IMAGE_NAME} **dbref**
```
Comment (rzueger) [inline devops/openshift/nice2_and_docker.rst]: NICE2_HIKARI_dataSource__sslMode=required

File: devops/openshift/nice2_and_docker.rst [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+Running the Image Locally
+-------------------------
+
+.. parsed-literal::
+
+    docker run --rm -p 8080:8080 -e NICE2_HIKARI_dataSource.serverName=\ **${DB_SERVER}**
+      -e NICE2_LOGBACK_CONFIG=\ **logback_terminal** -e NICE2_HIKARI_dataSource.databaseName=\ **${DB_NAME}** \
+      -e NICE2_HIKARI_dataSource.user=\ **${DB_USER}** -e NICE2_HIKARI_dataSource.password=\ **${DB_PASSWORD}** \
+      -e NICE2_JAVA_OPT\_-Dch.tocco.nice2.runenv=\ **development** -e NICE2_HIKARI_dataSource__sslMode=required \
```
Comment (rzueger) [inline devops/openshift/nice2_and_docker.rst]: `require` instead of `required`

**Comment by pgerber** [comment]
@rzueger, what exact command did you use to start it?

**Comment by pgerber** [comment]
@rzueger, I think I found it. Version isn't correctly set during start of Tocco.

**Pull request merged**